package.json

```diff
@@ -6,6 +6,10 @@
     "type": "git",
     "url": "https://github.com/w3c/reffy.git"
   },
+  "files": [
+    "index.js",
+    "src/"
```

Comment:
Won't `builds` be needed as well for `crawlSpecs`?

Also, since all `src/` script files will be in the pakage, why not include `reffy.js` as well?

Reply:
thanks, fixed